### PR TITLE
Associative array Generator

### PR DESCRIPTION
**Pull request by giorgiosironi**
Check out the `examples/` @dbellettini

### CODE REVIEW

File: src/Eris/Generator/SuchThatGenerator.php [hunk 1]
```diff
@@ -4,11 +4,17 @@
 use Eris\Generator;
 use LogicException;
 
+/**
```
Comment (MirkoBonadei) [inline src/Eris/Generator/SuchThatGenerator.php]: Probably this is not part of this pull request.

**Comment by MirkoBonadei** [comment]
:+1:

File: src/Eris/Generator/SuchThatGenerator.php (hunk 1 above)
Comment (giorgiosironi) [inline src/Eris/Generator/SuchThatGenerator.php]: Boy scout
